pkgs/development/python-modules/mbed-os-tools/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, buildPythonPackage, fetchPypi
+, appdirs, colorama, fasteners, future, intelhex, junit-xml
+, lockfile, prettytable, pyserial, requests, six
+}:
+
+buildPythonPackage rec {
+  pname = "mbed-os-tools";
+  version = "0.0.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "09v9a3hh6nfz7iv7c9wi6sjysck5z25mh1bq4bmikadnk227yh4s";
+  };
+
+  propagatedBuildInputs = [
+    appdirs
+    colorama
+    fasteners
+    future
+    intelhex
+    junit-xml
+    lockfile
+    prettytable
+    pyserial
+    requests
+    six
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = https://github.com/ARMmbed/mbed-os-tools;
+    description = "List processing tools and functional utilities";
+    license = licenses.apache2;
```

Comment:
```suggestion
    license = licenses.asl20;
```

to avoid:
```
error: while querying the derivation named 'mbed-cli-1.10.1':
while evaluating the attribute 'out.outPath' at /home/jon/.cache/nix-review/pr-68947/nixpkgs/lib/customisation.nix:151:13:
while evaluating the attribute 'propagatedBuildInputs' of the derivation 'mbed-cli-1.10.1' at /home/jon/.cache/nix-review/pr-68947/nixpkgs/pkgs/development/interpreters/python/mk-python-derivation.nix:105:3:
while evaluating the attribute 'out.outPath' at /home/jon/.cache/nix-review/pr-68947/nixpkgs/lib/customisation.nix:151:13:
while evaluating 'assertValidity' at /home/jon/.cache/nix-review/pr-68947/nixpkgs/pkgs/stdenv/generic/check-meta.nix:247:20, called from /home/jon/.cache/nix-review/pr-68947/nixpkgs/pkgs/stdenv/generic/make-derivation.nix:278:18:
while evaluating 'checkValidity' at /home/jon/.cache/nix-review/pr-68947/nixpkgs/pkgs/stdenv/generic/check-meta.nix:228:19, called from /home/jon/.cache/nix-review/pr-68947/nixpkgs/pkgs/stdenv/generic/check-meta.nix:248:18:
while evaluating anonymous function at /home/jon/.cache/nix-review/pr-68947/nixpkgs/pkgs/stdenv/generic/check-meta.nix:41:56, called from /home/jon/.cache/nix-review/pr-68947/nixpkgs/pkgs/stdenv/generic/check-meta.nix:231:13:
while evaluating 'toList' at /home/jon/.cache/nix-review/pr-68947/nixpkgs/lib/lists.nix:283:12, called from /home/jon/.cache/nix-review/pr-68947/nixpkgs/pkgs/stdenv/generic/check-meta.nix:42:71:
while evaluating the attribute 'meta.license' at /home/jon/.cache/nix-review/pr-68947/nixpkgs/pkgs/development/python-modules/mbed-os-tools/default.nix:34:5:
attribute 'apache2' missing, at /home/jon/.cache/nix-review/pr-68947/nixpkgs/pkgs/development/python-modules/mbed-os-tools/default.nix:34:15
```

Reply:
Thanks for such a quick review! :racing_car: 
Didn't expect that :sweat_smile:, fixing these and updating the packages atm